ops/lib/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 _libraries = {}  # type: Dict[Tuple[str,str], List[_Lib]]
 
-_libline_re = re.compile(r'''^LIB([A-Z]+)\s+=\s+([0-9]+|['"][a-zA-Z0-9_.-@]+['"])\s*$''')
+_libline_re = re.compile(r'''^LIB([A-Z]+)\s*=\s*([0-9]+|['"][a-zA-Z0-9_.\-@]+['"])''')
```

Comment:
I think it is common to put '-' as the last entry in [] which ends up meaning you don't have to escape it:
```
>>> re.match('([a-z_.-]*)', 'fo-o').groups()
('fo-o',)
>>> re.match('([a-z_.-@]*)', 'fo-o').groups()
('fo',)
```

Then again, I expect that was exactly what was done originally, and then someone thought to add '@' which broke what previously worked. :)